src/main/java/org/fcrepo/audit/InternalAuditor.java

```diff
@@ -231,7 +226,18 @@ public void createAuditNode(final FedoraEvent event) throws IOException {
     }
 
     @VisibleForTesting
+    protected String getEventPath(final String eventID) {
+        if (!eventID.startsWith("urn:uuid:")) {
+            throw new IllegalArgumentException("Event ID must be a 'urn:uuid:'" + eventID);
+        }
+        return pathMinter.get(eventID.substring("urn:uuid:".length()));
```

Comment:
Indeed, I have this related PR against this branch: https://github.com/acoburn/fcrepo4/pull/5
The reason I went towards extending `HierarchicalIdentifierSupplier` was because `fcrepo-audit-triplestore` will need the same logic.
But I would be happy to approach this differently as well.